src/EFCore.PG/Migrations/NpgsqlMigrationsSqlGenerator.cs

```diff
@@ -961,46 +961,35 @@ protected override void Generate(CreateSequenceOperation operation, IModel model
             string schema,
             string table,
             string name,
-            Type clrType,
-            string type,
-            bool? unicode,
-            int? maxLength,
-            bool? fixedLength,
-            bool rowVersion,
-            bool nullable,
-            object defaultValue,
-            string defaultValueSql,
-            string computedColumnSql,
-            IAnnotatable annotatable,
+            ColumnOperation operation,
```

Comment:
It looks much better now.

Reply:
Yeah, it does - but see [this comment from me](https://github.com/aspnet/EntityFrameworkCore/issues/14867#issuecomment-484870163) on a problem that it introduces.